crates/matrix-sdk-common/src/linked_chunk/mod.rs

```diff
@@ -1419,6 +1419,22 @@ impl EmptyChunk {
     }
 }
 
+/// The raw representation of a linked chunk, as persisted in storage.
```

Comment:
Maybe you can specifiy that this type is like `Chunk` except that ends/links to previous and next chunks are based on `ChunkIdentifier` instead of pointers. That's clearly an important detail.